litmus-portal/frontend/src/components/CustomResultModal/styles.ts

```diff
@@ -0,0 +1,87 @@
+import { makeStyles } from "@material-ui/core";
+
+export const useStyles = makeStyles((theme) => ({
+	testHeading: {
+		marginTop: theme.spacing(6.25),
+		fontFamily: "Ubuntu",
+		fontSize: 25,
+	},
+	testType: {
+		fontSize: 17,
+		fontFamily: "Ubuntu",
+		paddingRight: theme.spacing(1.25),
+	},
+	testResult: {
+		color: "#109B67",
+		fontSize: 17,
+		fontFamily: "Ubuntu",
```

Comment:
Can you override the default font to be Ubuntu, so that you don't need to mention it everywhere.
You can override it in the theme data like so: 
https://material-ui.com/customization/typography/

Reply:
- [x] Added the font in the main theme data